angular/projects/spark-angular/src/lib/components/sprk-stack/sprk-stack.stories.ts

```diff
@@ -371,25 +333,27 @@ export const stackSplitLayoutMixed = () => ({
     <sprk-stack splitAt="tiny">
       <div
         sprkStackItem
-        class="sprk-o-Stack__item--fourth@xs sprk-u-AbsoluteCenter"
+        class="sprk-o-Stack sprk-o-Stack__item--fourth@xs sprk-o-Stack--center-all"
       >
-        <p class="sprk-b-TypeBodyOne">
+        <p sprkStackItem class="sprk-b-TypeBodyOne">
           fourth
         </p>
       </div>
+
       <div
         sprkStackItem
-        class="sprk-o-Stack__item--half@xs sprk-u-AbsoluteCenter"
+        class="sprk-o-Stack sprk-o-Stack__item--half@xs sprk-o-Stack--center-all"
```

Comment:
Angular could not use the Stack component here and had to use stack added as additional classes due to the custom elements that angular applies. This is for the muli-nested stack example so okay for now. We will need to take a look at this in a future issue though as it might come up in the cases of complex nested stacks.